debian/control

```diff
@@ -43,27 +52,40 @@ Depends: ${misc:Depends},
          python3-pkg-resources,
          ${extra:Depends}
 Recommends: ubuntu-pro-client-l10n
-Description: management tools for Ubuntu Pro
- Ubuntu Pro is the professional package of tooling, technology
- and expertise from Canonical, helping organisations around the world
- manage their Ubuntu deployments.
- .
- Subscribers to Ubuntu Pro will find helpful tools for accessing
- services in this package.
+Breaks: ubuntu-advantage-tools (<<31~)
+Replaces: ubuntu-advantage-tools (<<31~)
+# IMPORTANT: ubuntu-pro-client does not "Provide" ubuntu-advantage-tools
+# TODO fill this in with rationale after testing.
```

Comment:
I think it's fine not to have a Provides for the replaced package if not needed, but if you have an explanation on why we don't want it (so we avoid adding it in the future), +1 for adding a comment on it under the ubuntu-pro-client binary package stanza.